gallery/src/components/NxDropdownIconOnly/NxDropdownIconOnlyLinksExample.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { NxDropdownIconOnly, useToggle, NxTextLink } from '@sonatype/react-shared-components';
+
+function NxDropdownIconOnlyActionsExample() {
+  const [isOpen, onToggleCollapse] = useToggle(false),
+      deleteFn = () => { alert('delete'); },
+      onClick = () => { alert('click'); };
+
+  return (
+    <NxDropdownIconOnly icon={faCog}
+                        isOpen={isOpen}
+                        onToggleCollapse={onToggleCollapse}
+                        title="Links options">
+      <NxTextLink external className="nx-dropdown-link disabled">Save</NxTextLink>
+      <NxTextLink external className="nx-dropdown-link" onClick={deleteFn}>Delete</NxTextLink>
+      <NxTextLink external className="nx-dropdown-link" onClick={onClick}>Some Action</NxTextLink>
```

Comment:
It seems a little unrealistic that links with names like "Save" and "Delete" would be external

Reply:
> It seems a little unrealistic that links with names like "Save" and "Delete" would be external

I just wanted the icons, but I have it in the last one: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/4784219f8158eff30b7784caad8e5da31a174bb2